.github/workflows/ci.yml

```diff
@@ -164,6 +172,15 @@ jobs:
           # build docs
           cargo doc --target ${{ matrix.target }} -p wgpu --no-deps
 
+      - name: check web-em
+        if: matrix.kind == 'webgl-em'
+        run: |
+          # check with no features
+          cargo ${{matrix.tool}} --target ${{ matrix.target }} -p wgpu -p wgpu-core
```

Comment:
I think we need to do the same as regulalr webgl - require `webgl` feature to be passed in. Otherwise, this will be confusing when we get emscripten webgpu target.